WordPress/Classes/ViewRelated/Me/App Settings/About/UnifiedAboutViewController.swift

```diff
@@ -99,12 +96,16 @@ class UnifiedAboutViewController: UIViewController, OrientationLimited {
 
     // MARK: - View lifecycle
 
-    static func controller(configuration: AboutScreenConfiguration) -> UIViewController {
-        let controller = UnifiedAboutViewController(configuration: configuration)
+    static func controller(appInfo: AboutScreenAppInfo? = nil, configuration: AboutScreenConfiguration, fonts: AboutScreenFonts? = nil) -> UIViewController {
```

Comment:
nit: since this method returns a navigation controller with the current UIViewcontroller as root, can we make the name of this method a bit clearer? It sounds like it's returning an instance of the vc itself.